### PR TITLE
CVE records for May 5th

### PR DESCRIPTION
**Pull request by monicamurray**

### CODE REVIEW

File: 2021/27xxx/CVE-2021-27759.json [hunk 1]
```diff
@@ -1,18 +1,87 @@
 {
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27759.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/27xxx/CVE-2021-27759.json:1]: ('Invalid control character at: line 5 column 32 (char 144)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27759.json (hunk 1 above)
Comment (monicamurray) [reply 2021/27xxx/CVE-2021-27759.json:1]: I modified that line and am trying a new git push...

Comment (cve-team) [inline 2021/27xxx/CVE-2021-27759.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/27xxx/CVE-2021-27759.json:1]: ('Expecting value: line 50 column 33 (char 1718)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27759.json (hunk 1 above)
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27759.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/27xxx/CVE-2021-27759.json:1]: ('Expecting value: line 55 column 38 (char 1918)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27759.json (hunk 1 above)
Comment (monicamurray) [reply 2021/27xxx/CVE-2021-27759.json:1]: corrected line 55

Comment (anthnysingleton) [reply 2021/27xxx/CVE-2021-27759.json:1]: Looks like there is some weird char at the beginning of this file before the { bracket.

File: 2021/27xxx/CVE-2021-27759.json [hunk 2]
```diff
@@ -1,18 +1,87 @@
...
+        "engine": "Vulnogram 0.0.9"
+    },
+    "impact": {
+        "cvss": {
+            "attackComplexity": "HIGH",
+            "attackVector": "ADJACENT_NETWORK",
+            "availabilityImpact": "NONE",
+            "baseScore": 1.6,
+            "baseSeverity": "LOW",
+            "confidentialityImpact": “LOW”,
```
Comment (anthnysingleton) [inline 2021/27xxx/CVE-2021-27759.json]: The qoutes for the values of confidentialityImpact, privilegesRequired, and userInteraction are not accepted. This usually happens from a copy/paste. Just remove quotes and reapply quotes in your editor.

**Comment by monicamurray** [comment]
I see a message showing cve-team requested changes, but I do not see any comments about the change needed.  Please post any additional errors found.

**Comment by anthnysingleton** [comment]
I posted two comments last night. Once you resolve the issues in [2021/27xxx/CVE-2021-27759.json](https://github.com/CVEProject/cvelist/pull/5613/files/a64422dbf5b96f373d48ba9f02d2219ba6cfd9d5#diff-be82be3abe8d51b38ae5775ceb78ec13f4c116605491afd735e1894a95c66cf3) then PR will continue to be processed. We are close to getting your first PR accepted =] 


Just replace the CVE-2021-27759.json file contents with this:

```
{
    "CVE_data_meta": {
        "ASSIGNER": "PSIRT@hcl.com",
        "DATE_PUBLIC": "2022-04-28T00:00:00.000Z",
        "ID": "CVE-2021-27759",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "product_name": "HCL BigFix Inventory",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "9.x"
                                        },
                                        {
                                            "version_value": "10.x"
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    "vendor_name": "HCL Software"
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "This vulnerability arises because the application allows the user to perform some sensitive action without verifying that the request was sent intentionally. An attacker can cause a victim's browser to emit an HTTP request to an arbitrary URL in the application.\n"
            }
        ]
    },
    "generator": {
        "engine": "Vulnogram 0.0.9"
    },
    "impact": {
        "cvss": {
            "attackComplexity": "HIGH",
            "attackVector": "ADJACENT_NETWORK",
            "availabilityImpact": "NONE",
            "baseScore": 1.6,
            "baseSeverity": "LOW",
            "confidentialityImpact": "LOW",
            "integrityImpact": "NONE",
            "privilegesRequired": "LOW",
            "scope": "UNCHANGED",
            "userInteraction": "REQUIRED",
            "vectorString": "CVSS:3.1/AV:A/AC:H/PR:L/UI:R/S:U/C:L/I:N/A:N/E:F/RL:U/RC:C/CR:L/IR:L/AR:L/MAV:A/MAC:H/MPR:L/MUI:R/MS:U/MC:X/MI:N/MA:N",
            "version": "3.1"
        }
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-352 Cross-Site Request Forgery (CSRF)"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "refsource": "CONFIRM",
                "url": "https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0098006"
            }
        ]
    },
    "source": {
        "discovery": "UNKNOWN"
    }
}
```

**Comment by monicamurray** [comment]
ok, I think I fixed the errors.  Fingers crossed!

**Pull request merged**